lib/sync_checker/formats/publication_check.rb

```diff
@@ -40,7 +40,8 @@ def checks_for_live(locale)
         ]
       end
 
-      def document_type
+      def document_type(edition)
```

Comment:
Can we just always return `edition.publication_type.key`? I'm not sure we need  line 45 as we're always passing the edition that we're checking